pages/about/jobs.md

```diff
@@ -0,0 +1,26 @@
+---
+title: .Gov jobs
+permalink: /get-gov/jobs/
+layout: layouts/info-page
+sidenav: true
+excerpt: Work with the .gov team.
+tags: jobs
+eleventyNavigation:
+  key: data
```

Comment:
Same for the `key` and for `tags` here, if want them to appear in collection with `about` we need them to have the same value as the about page's `key` and `tags`. `tag` will group them together as a collection, and `key` determines which sidenav it will be displayed in

Reply:
Fixed